backend/level-2/exercise.spec.ts

```diff
@@ -23,11 +24,22 @@ app.get('/', (req: unknown, res: unknown) => {
  * - Define a route `/protected` that returns the value of the 'x-shadowy-user' header.
  */
 const authMiddleware = (req: unknown, res: unknown, next: unknown) => {
-    // TODO: complete this function
+    const myReq = req as express.Request
+    const myRes = res as express.Response
+    const myNext = next as Function
+    if(!myReq.headers['x-shadowy-user']){
+        myRes.status(403).send()
+    } else {
+        myRes.statusCode = 200
+        myNext()        
+    }
 }
 
 app.get('/protected', authMiddleware,  async (req: unknown, res: unknown) => {
-    // TODO: complete this function
+    const myReq = req as express.Request
+    const myRes = res as express.Response 
```

Comment:
This is wrong. Think of another way to define this types without using `as`.

Tip: you can update everything except the tests